pipeline-model-definition/src/test/resources/json/whenBeforeInputTrue.json

```diff
@@ -1,54 +1,61 @@
 {"pipeline": {
-  "stages":   [
-    {
-      "name": "One",
-      "branches": [      {
-        "name": "default",
-        "steps": [        {
-          "name": "echo",
-          "arguments": [          {
-            "key": "message",
-            "value":             {
-              "isLiteral": true,
-              "value": "Hello"
-            }
-          }]
-        }]
-      }]
-    },
-    {
-      "name": "Two",
-      "branches": [      {
-        "name": "default",
-        "steps": [        {
-          "name": "script",
-          "arguments": [          {
-            "key": "scriptBlock",
-            "value":             {
-              "isLiteral": true,
-              "value": "echo \"World\"\n                    echo \"Heal it\""
-            }
-          }]
+  "stages": [  {
+    "name": "One",
+    "when":     {
+      "conditions": [      {
+        "name": "expression",
+        "arguments": [        {
+          "key": "scriptBlock",
+          "value":           {
+            "isLiteral": true,
+            "value": "false"
```

Comment:
So this file should be renamed to `whenBeforeInputFalse.json` and `AbstractModelDefTest`'s list of test cases should be changed accordingly - since this is the JSON form of `whenBeforeInputFalse.groovy`, not `whenBeforeInputTrue.groovy`.

Reply:
This JSON has been changed, now is really beforeInputTrue

Reply:
No, this is still wrong - the `when` condition here is returning false, as in `whenBeforeInputFalse.groovy`. As I said above, this file should be renamed to `whenBeforeInputFalse.json` and `whenBeforeInputFalse` should be used in `AbstractModelDefTest`, since `whenBeforeInputFalse` will not hang in the execution of converted tests, while `whenBeforeInputTrue` will hang.